package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "welcome",
+  "version": "1.0.0",
+  "description": "a series of probot plugins that welcome new users to your repository",
+  "main": "index.js",
+  "scripts": {
+    "start": "probot run"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/behaviorbot/welcome.git"
+  },
+  "author": "hiimbex",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/behaviorbot/welcome/issues"
+  },
+  "homepage": "https://github.com/behaviorbot/welcome#readme",
+  "dependencies": {
+    "probot": "^0.8.0",
+    "first-pr-merge": "behaviorbot/first-pr-merge",
+    "new-pr-welcome": "behaviorbot/new-pr-welcome",
+    "new-issue-welcome": "behaviorbot/new-issue-welcome#fix-name"
```

Comment:
FIXME: replace after https://github.com/behaviorbot/new-issue-welcome/pull/25 is merged